server/container/guice/guice-common/src/test/java/org/apache/james/MailsShouldBeWellReceived.java

```diff
@@ -89,6 +94,45 @@ default void mailsShouldBeWellReceived(GuiceJamesServer server) throws Exception
 
     }
 
+    @Test
+    default void mailsShouldBeWellReceivedByBothRecipient(GuiceJamesServer server) throws Exception {
+        server.getProbe(DataProbeImpl.class).fluent()
+            .addDomain(DOMAIN)
+            .addUser(JAMES_USER, PASSWORD)
+            .addUser(OTHER_USER, PASSWORD_OTHER);
+
+        MailboxProbeImpl mailboxProbe = server.getProbe(MailboxProbeImpl.class);
+        mailboxProbe.createMailbox("#private", JAMES_USER, DefaultMailboxes.INBOX);
+        mailboxProbe.createMailbox("#private", OTHER_USER, DefaultMailboxes.INBOX);
+
+        Port smtpPort = server.getProbe(SmtpGuiceProbe.class).getSmtpPort();
+        String message = Resources.toString(Resources.getResource("eml/htmlMail.eml"), StandardCharsets.UTF_8);
+
+        try (SMTPMessageSender sender = new SMTPMessageSender(Domain.LOCALHOST.asString())) {
+            Mono.fromRunnable(
+                Throwing.runnable(() -> {
+                    sender.connect(JAMES_SERVER_HOST, smtpPort);
+                    sendUniqueMessageToTwoUsers(sender, message);
+                }))
+                .subscribeOn(Schedulers.elastic())
```

Comment:
It's it(`.subscribeOn(Schedulers.elastic())`) necessary?

Reply:
The whole Mono is probably useless